03week/ticTacToe.js

```diff
@@ -23,26 +23,66 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function horizontalWin() {
-  // Your code here
+// Check for winning horizontal with if
+// 00 01 02
+// 10 11 12
+// 20 21 22
+const horizontalWin = () => {
+  if ((board[0][0] === board[0][1] && board[0][1] === board[0][2]) || (board[1][0] === board[1][1] && board[1][1] === board[1][2]) || (board[2][0] === board[2][1] &&
+      board[2][1] === board[2][2])) {
+    return true;
+  }
 }
 
-function verticalWin() {
-  // Your code here
+//Check for winning vertical with if
+//00 10 20
+//01 11 21
+//02 12 22
+const verticalWin = () => {
+  if ((board[0][0] === board[1][0] && board[1][0] === board[2][0]) || (board[0][1] === board[1][1] && board[1][1] === board[2][1]) || (board[0][2] === board[1][2] &&
+      board[1][2] === board[2][2])) {
+    return true;
+  }
 }
 
-function diagonalWin() {
-  // Your code here
+// Check for winning diagonalWin with if
+// 00 11 22
+// 02 11 20
+const diagonalWin = () => {
+  if (board[0][0] === board[1][1] && board[1][1] === board[2][2] || (board[0][2] === board[1][1] && board[1][1] === board[2][0])) {
+    return true;
+  }
 }
 
-function checkForWin() {
-  // Your code here
+// Got help from eddy, I do not completely understand yet
+const checkForWin = () => {
+  // if (horizontalWin() || verticalWin() || diagonalWin()) {
```

Comment:
You can do it! This logic is correct, keep going! Just ignore the turncount thing, and if you look at the tests below, they expect checkForWin() to return either true or false.